src/modules/UsersPluginKerberos.pm

```diff
@@ -150,7 +150,16 @@ sub PluginPresent {
 	y2debug ("Kerberos plugin not present");
 	return 0;
     }
-    my $out	= SCR->Execute (".target.bash_output", "/usr/lib/mit/sbin/kadmin.local -nq 'list_principals ".String->Quote("".$data->{uid})."*' | /usr/bin/grep '".String->Quote("".$data->{uid})."*'");
+
+    # Note that the last kbr5 package provides the kadmin.local binary at /usr/sbin,
+    # but older kbr5 uses /usr/lib/mit/sbin path.
+
+    my $out = SCR->Execute (".target.bash_output", "/usr/sbin/kadmin.local -nq 'list_principals ".String->Quote("".$data->{uid})."*' | /usr/bin/grep '".String->Quote("".$data->{uid})."*'");
+
+    if ($out->{"stdout"} =~ /^$data->{uid}/ ) {
```

Comment:
this looks kind of strange to check stdout...why not exit code or stderr? Exit code is 127 if command not found.

Reply:
The same as below. Why not check for the tool first, then run it.